test/event-docs.json

```diff
@@ -0,0 +1,1930 @@
+            },
+            "rcpt_type": {
+              "description": "Indicates that a recipient address appeared in the Cc or Bcc header or the archive JSON array",
+              "sampleValue": "cc"
+            },
+            "raw_reason": {
+              "description": "Unmodified, exact response returned by the remote server due to a failed delivery attempt",
+              "sampleValue": "MAIL REFUSED - IP (17.99.99.99) is in black list"
+            },
+            "reason": {
+              "description": "Canonicalized text of the response returned by the remote server due to a failed delivery attempt",
+              "sampleValue": "MAIL REFUSED - IP (a.b.c.d) is in black list"
+            },
+            "routing_domain": {
+              "description": "Domain receiving this message",
+              "sampleValue": "example.com"
+            },
+            "sending_ip": {
+              "description": "IP address through which this message was sent",
+              "sampleValue": "127.0.0.1"
+            },
+            "sms_coding": {
+              "description": "Data encoding used in the SMS message",
+              "sampleValue": "ASCII"
+            },
+            "sms_dst": {
+              "description": "SMS destination address",
+              "sampleValue": "7876712656"
+            },
+            "sms_dst_npi": {
+              "description": "Destination numbering plan identification",
+              "sampleValue": "E164"
+            },
+            "sms_dst_ton": {
+              "description": "Type of number for the destination address",
+              "sampleValue": "International"
+            },
+            "sms_src": {
+              "description": "SMS source address",
+              "sampleValue": "1234"
+            },
+            "sms_src_npi": {
+              "description": "Source numbering plan identification",
+              "sampleValue": "E164"
+            },
+            "sms_src_ton": {
+              "description": "Type of number for the source address",
+              "sampleValue": "Unknown"
+            },
+            "subaccount_id": {
+              "description": "Unique subaccount identifier.",
+              "sampleValue": "101"
+            },
+            "subject": {
+              "description": "Subject line from the email header",
+              "sampleValue": "Summer deals are here!"
+            },
+            "template_id": {
+              "description": "Slug of the template used to construct this message",
+              "sampleValue": "templ-1234"
+            },
+            "template_version": {
+              "description": "Version of the template used to construct this message",
+              "sampleValue": "1"
+            },
+            "timestamp": {
+              "description": "Event date and time, in Unix timestamp format (integer seconds since 00:00:00 GMT 1970-01-01)",
+              "sampleValue": "1454442600"
```

Comment:
[This](http://spock.trymsys.net/api/v1/webhooks/events/documentation) Momentum instance returns:

```
"relay_permfail": {
                    "event": {
                                 :
                                 :
"timestamp": {
                            "description": "Event date and time, in Unix timestamp format (integer seconds since 00:00:00 GMT 1970-01-01)",
                            "sampleValue": 1427736822
                        },
                                 :
                                 :
                     }
```

But this test uses a quoted number for `sampleValue`. I think that is a bug in the documentation that was fixed at some point. Is it worth handling both formats?